.semaphore/semaphore.yml

```diff
@@ -84,6 +84,7 @@ blocks:
               values:
                 - guava-with-google-checks
                 - guava-with-sun-checks
+                - openjdk14-with-checks-nonjavadoc-error
```

Comment:
@pbludov , this is not very good approach for CI config.

it should be:
`.ci/no-exception-test.sh openjdk14-with-checks-nonjavadoc-error`

and in comamnds we do `eval $COMMAND`

one day this execution will fail in CI in PR, contributor might not be very experienced in CI to figureout how to reproduce it on local, so we will share with him link to single line in config for simple copy and paste in terminal, and it works.

Reply:
done in https://github.com/checkstyle/checkstyle/pull/8950